File: go/scraper.php
<?php
echo "hello world!!";

$url = "https://show.1688.com/pinlei/industry/pllist.html?spm=a260j.12536015.jr60bfo3.1.10f3700ecaYFQt&&sceneSetId=857&sceneId=2158&bizId=4404";
$ch = curl_init();
curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_HEADER, TRUE);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
$data = curl_exec($ch);
$httpCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
curl_close($ch);

print_r($data);

File: go/scraper.go
package main

import (
	"fmt"
	"io/ioutil"
	"net/http"
)

func main() {
	url := "https://show.1688.com/pinlei/industry/pllist.html?spm=a260j.12536015.jr60bfo3.1.10f3700ecaYFQt&&sceneSetId=857&sceneId=2158&bizId=4404"
	req, _ := http.NewRequest("GET", url, nil)
	res, _ := http.DefaultClient.Do(req)
	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)
	fmt.Println(string(body))
}
